app/recordtransfer/tests/unit/test_models.py

```diff
@@ -98,6 +108,72 @@ def test_new_session_creation(self) -> None:
         self.assertIsInstance(self.session, UploadSession)
         self.assertEqual(self.session.status, UploadSession.SessionStatus.CREATED)
         self.assertEqual(len(self.session.token), 32)
+        self.assertIsNotNone(self.session.last_upload_interaction_time)
+
+    @patch("django.conf.settings.UPLOAD_SESSION_EXPIRE_AFTER_INACTIVE_MINUTES", 30)
+    def test_expires_at(self) -> None:
+        """Test expires_at returns the correct expiration time for sessions in CREATED and
+        UPLOADING states, and None for other states.
+        """
+        fixed_now = datetime(2025, 2, 18, 12, 0, 0)
+
+        # Test CREATED state
+        self.session.status = UploadSession.SessionStatus.CREATED
+        self.session.last_upload_interaction_time = fixed_now
+        expected_expires_at = self.session.last_upload_interaction_time + timezone.timedelta(
+            minutes=30
+        )
+        self.assertEqual(self.session.expires_at, expected_expires_at)
+
+        # Test UPLOADING state
+        self.session.status = UploadSession.SessionStatus.UPLOADING
+        self.session.last_upload_interaction_time = fixed_now
+        expected_expires_at = self.session.last_upload_interaction_time + timezone.timedelta(
+            minutes=30
+        )
+        self.assertEqual(self.session.expires_at, expected_expires_at)
```

Comment:
Is this a valid simplification?

```suggestion
        valid_states = [
            UploadSession.SessionStatus.CREATED,
            UploadSession.SessionStatus.UPLOADING,
        ]

        self.session.last_upload_interaction_time = fixed_now
        expected_expires_at = fixed_now + timezone.timedelta(minutes=30)

        for state in valid_states:
            self.session.status = state
            self.assertEqual(self.session.expires_at, expected_expires_at)
```